tests/app/scheduler/modules/random_channel_test.py

```diff
@@ -12,13 +12,13 @@ def test_doing_it(bot, config, app):
     config.slack_notification_channel = ''
     config.slack_announcement_channel = ''
     bot.get_channels.return_value = [{'id': '123', 'name': 'general'},
-                                     {'id': '321', 'name': 'random'}]
+                                     {'id': '321', 'name': 'random', 'is_archived': True},
+                                     {'id': '456', 'name': 'bobheadxi', 'is_private': True}]
```

Comment:
These 2 lines are too long. We are having a bit of trouble with the CI at the moment, so bare with us.

You can run the tests that our CI runs with `scripts/build_check.sh` in your terminal.